src/components/ColorPicker/ColorPicker.scss

```diff
@@ -13,9 +13,9 @@ $inner-shadow: inset 0 0 2px 0 rgba(0, 0, 0, 0.5);
 }
 
 .ColorPicker {
-  --pc-color: 10;
-  --pc-adjustments: 20;
-  --pc-dragger: 30;
+  --pc-color-picker-color: 10;
```

Comment:
Maybe this should be called sheet or background. It's not really `color`. Any thoughts @kyledurand ?

Reply:
Oh yeah true. This node sets the root stacking context so I'm just going to go with `--pc-color-picker`